tests/test_backport.py

```diff
@@ -346,3 +346,32 @@ def __init__(self, *, getitem=None, delete=None, post=None):
     gh = FakeGH(getitem=getitem)
     await backport.router.dispatch(event, gh)
     assert len(gh.post_) == 0
+
+
+@pytest.mark.parametrize('ref', ['3.9', '4.0', '3.10'])
```

Comment:
There is no current plan for 4.0, certainly not 4.0 *and* 3.10.  I believe a majority agreed on 3.10 *instead* of 4.0, but a few seem to support 4.0 rather than 3.10.  We probably should revisit this soon.

Reply:
`4.0` is still a valid test case.